packages/jellyfish-transaction/__tests__/script/dftx/dftx_masternode/CreateMasterNode.test.ts

```diff
@@ -20,85 +22,35 @@ it('should bi-directional buffer-object-buffer', () => {
   })
 })
 
-describe('without `operatorPubKeyHash`', () => {
-  const header = '6a1a4466547843' // OP_RETURN (length 26 = 0x1a), PUSH_DATA(44665478, 43)
-  const data = '01742b337e0f40d5f229a89d3a26d53ae1093b6cff'
-  const createMasterNode: CreateMasterNode = {
-    type: 0x01,
-    collateralPubKeyHash: '742b337e0f40d5f229a89d3a26d53ae1093b6cff'
-  }
-
-  it('should craft dftx with OP_CODES._()', () => {
-    const stack = [
-      OP_CODES.OP_RETURN,
-      OP_CODES.OP_DEFI_TX_CREATE_MASTER_NODE(createMasterNode)
-    ]
-
-    const buffer = toBuffer(stack)
-    expect(buffer.toString('hex')).toStrictEqual(header + data)
-  })
-
-  describe('Composable', () => {
-    it('should compose from buffer to composable', () => {
-      const buffer = SmartBuffer.fromBuffer(Buffer.from(data, 'hex'))
-      const composable = new CCreateMasterNode(buffer)
-      expect(composable.toObject()).toStrictEqual(createMasterNode)
-    })
-
-    it('should compose from composable to buffer', () => {
-      const composable = new CCreateMasterNode(createMasterNode)
-      const buffer = new SmartBuffer()
-      composable.toBuffer(buffer)
+const header = '6a1a4466547843' // OP_RETURN(0x1a) (length 26 = 0x1a) CDfTx.SIGNATURE(0x44665478) CCreateMasterNode.OP_CODE(0x43)
+const data = '01742b337e0f40d5f229a89d3a26d53ae1093b6cff' // CreateMasterNode.operatorType(0x01) CreateMasterNode.operatorAuthAddress(0x742b337e0f40d5f229a89d3a26d53ae1093b6cff)
+const createMasterNode: CreateMasterNode = {
+  operatorType: 0x01,
+  operatorAuthAddress: '742b337e0f40d5f229a89d3a26d53ae1093b6cff'
```

Comment:
hmm... for txn test, instead of adding another describe block..
its sufficient to extend the hex on test as serialization doesn't have vary business logics (testing the same condition only)

https://github.com/DeFiCh/jellyfish/blob/703d059a2ab08f3b52fc7dc29996661f717dd243/packages/jellyfish-transaction/__tests__/script/dftx/dftx_masternode/CreateMasterNode.test.ts#L12-L13

https://github.com/DeFiCh/jellyfish/blob/703d059a2ab08f3b52fc7dc29996661f717dd243/packages/jellyfish-transaction/__tests__/script/dftx/dftx_masternode/CreateMasterNode.test.ts#L26-L32